src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -33,4 +33,8 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+	public String getFeedbackToUser() {
+		return feedbackToUser;
+	}
+
```

Comment:
Is this for T3A2? Try not to include multiple different activities in a PR unless stated.